src/control/cmd/daos/snapshot.go

```diff
@@ -233,13 +233,14 @@ func (cmd *containerSnapListCmd) Execute(args []string) error {
 		return cmd.OutputJSON(snaps, nil)
 	}
 
-	cmd.Info("Container's snapshots :")
+	cmd.Infof("%s\t\t\t%s\t\t\t%s", "Timestamp" , "Epoch", "Name")
+	cmd.Infof("---------\t\t\t-----\t\t\t----")
 	if len(snaps) == 0 {
 		cmd.Info("no snapshots")
 		return nil
 	}
 	for _, snap := range snaps {
-		cmd.Infof("0x%x %s", snap.Epoch, snap.Name)
+		cmd.Infof("%s\t0x%x\t%s", snap.Timestamp, snap.Epoch, snap.Name)
```

Comment:
What does the JSON output look like? Is there any processing that needs to happen to make that output useful?

Reply:
The json output looks good and already has the timestamp, epoch and name.